src/components/Homepage/home.css

```diff
@@ -0,0 +1,373 @@
+* {
```

Comment:
Format this file.
Also, make it two spacing instead of 4.
You can also set your editor to 2 spacing by default